charts/ingress-nginx/templates/controller-deployment.yaml

```diff
@@ -180,6 +180,9 @@ spec:
         {{- if .Values.controller.resources }}
           resources: {{ toYaml .Values.controller.resources | nindent 12 }}
         {{- end }}
+        {{- if and .Values.controller.resizePolicy (semverCompare ">=1.30.0-0" .Capabilities.KubeVersion.Version) }}
```

Comment:
Please have a look how we are implementing such capability checks elsewhere in the chart and align yours to these.